proposals/acp/wac-acp-diff-story.md

```diff
@@ -1,5 +1,11 @@
 # ACP Compared to Web Access Control
 
+This document is intended to reflect the following authoritative-ish definitions:
+- [W3C wiki](http://www.w3.org/wiki/WebAccessControl)
+- [solid/specification](http://github.com/solid/specification)
+- [sold/web-access-control](http://github.com/solid/solid-spec)
```

Comment:
```suggestion
This document is intended to reflect the following authoritative-ish definitions/clarifications e.g.:
- [W3C wiki](http://www.w3.org/wiki/WebAccessControl)
- [solid/specification](http://github.com/solid/specification)
- [solid/solid-spec](http://github.com/solid/solid-spec)
- [solid/web-access-control-spec](http://github.com/solid/web-access-control-spec)
```

Reply:
> `This document is intended to reflect the following authoritative-ish definitions/clarifications e.g.:`

`the following` fights with `e.g.` (readable as, `for example`)

I suggest either --
`This document is intended to reflect the following authoritative-ish definitions/clarifications:`
-- or --
`This document is intended to reflect a number of authoritative-ish definitions/clarifications, including the following:`